build-python-host-emscripten.sh

```diff
@@ -22,16 +22,21 @@ pushd ../usr/local
 # includes aren't need for distribution, various libraries
 # won't be used in the web (at least for now)
 rm -rf bin include lib/pkgconfig lib/libpython3.11d.a
-rm -rf lib/python3.11/test
+rm -rf lib/python3.11/_osx_support.py
 rm -rf lib/python3.11/config-3.11d
```

Comment:
While you are touching, let's remove non-debug config dirs, too. The suffix depends on version and build flavor.

```suggestion
rm -rf lib/python3.11/config-3.*
```